Gemfile.lock

```diff
@@ -10,8 +10,11 @@ GEM
     addressable (2.8.5)
       public_suffix (>= 2.0.2, < 6.0)
     ast (2.4.2)
+    byebug (11.1.3)
     crack (0.4.5)
       rexml
+    defra_ruby_style (0.3.0)
```

Comment:
update defra_ruby_style to most recent version 0.4.0 
That has all the recent changes